### PR TITLE
title and sub-header now aligned properly

### PR DESCRIPTION
**Pull request by mr-mr**
Small tweaks to the main video title and stand first, now all aligned (hopefully)

gif-keyboard-6922171368322717703

### CODE REVIEW

**Comment by duarte** [comment]
Not really in these commits, but why does .docs--standfirst-shortdesc need to have a fixed height of 50px?

File: src/css/_docs.scss [hunk 1]
```diff
@@ -1,29 +1,35 @@
+$mobile-large:  640px;
```
Comment (duarte) [inline src/css/_docs.scss]: where will we use this?

Comment (mr-mr) [inline src/css/_docs.scss]: So I was looking at a weird half breakpoint, but Ben said something more stable might be coming in Phablet form. Wasn't a huge issue. Will remove for now.

I'll remove the fixed height.

**Comment by mr-mr** [comment]
gif-keyboard-164776072658681797

**Comment by gidsg** [comment]
👍 📦

**Comment by mr-mr** [comment]
:shipit: ?

**Pull request merged**